cmd/kube-apiserver/app/server.go

```diff
@@ -110,7 +111,6 @@ type APIServer struct {
 	MasterServiceNamespace     string
 	RuntimeConfig              util.ConfigurationMap
 	KubeletConfig              kubeletclient.KubeletClientConfig
-	ClusterName                string
```

Comment:
Yes. Have pulled out the change in https://github.com/kubernetes/kubernetes/pull/18388.
Will rebase once that is merged.